omero/sysadmins/public.txt

```diff
@@ -0,0 +1,135 @@
+Public data in the repository
+=============================
+
+The OMERO.web framework allows raw data to be published using built-in tools
+or supplied through webservices to the external web resources. This feature is
+available only after granting access to the selected datasets and making them
+visible to the 'public user', ensuring you always have control over access to
+your data.
+
+There are three ways of publishing data using OMERO.web:
+
+- using a URL to launch the web-based Image viewer, as described in
+  :ref:`launching_web_viewer`, which can be accompanied by a thumbnail. For
+  more details of how to load the thumbnail see
+  :ref:`urls_from_within_OMERO_web`
+
+- embedding the image viewport directly into other web resources, for more
+  details see :ref:`embedding_web_viewport`
+
+- allowing public access to the OMERO.web data manager
+
+The sections below describe how to set up these features.
+
+Public user
+-----------
+
+The OMERO.web framework supports auto-login for a single username / password.
+This means that any public visitors to certain OMERO.web pages will be
+automatically logged in and will be able to access the data available to the
+defined 'public user'.
+
+To set this up on your OMERO.web installation:
+
+- Create a group with read-only permissions (the name can be anything e.g.
+  "public-data"). We recommend read-only permissions so that the public user
+  will not be able to modify, delete or annotate data belonging to other
+  members.
+
+- Create a member of this group, noting the username and password (you will
+  enter these below). Again, the First Name, Last Name, username and
+  password can be anything you like.
+
+  .. note:: If you add this member to other groups, all data in these groups
+      will also become publicly accessible for as long as this user remains
+      in the group.
+
+- Enable the :property:`omero.web.public.enabled` property and set
+  :property:`omero.web.public.user` and
+  :property:`omero.web.public.password`:
+
+  ::
+
+     $ bin/omero config set omero.web.public.enabled True
+
+     $ bin/omero config set omero.web.public.user '<username>'
+
+     $ bin/omero config set omero.web.public.password '<password>'
+
+- Set the :property:`omero.web.public.url_filter`. This filter is a
+  regular expression that will allow only matching URLs to be accessed
+  by the public user.
+
+  There are three common use cases for the URL filter:
```

Comment:
This might be a bit confusing for some: Above we have "three ways of publishing data using OMERO.web" which quite closely match the "three common use cases for the URL filter" except for the second option. Above, the second option is "embedding the image viewport directly into other web resources" whereas the second option for the url_filter is an internal app "my_web_public". Either we need to make these two separate lists into one with 4 options, providing the url_filters alongside the publishing options. Or we keep the 2 lists but there is a 4th publishing option that describes an internal app E.g. "my_web_public".